source-code/ide-extension/src/services/telemetry/implementation.ts

```diff
@@ -60,14 +60,19 @@ async function capture(args: CaptureEventArguments) {
  * Gets the git origin url of the currently opened repository.
  */
 export async function getGitOrigin() {
-	const workspaceRoot = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath
-	const remotes = await raw.listRemotes({
-		fs,
-		dir: await raw.findRoot({
+	try {
+		const workspaceRoot = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath
+		const remotes = await raw.listRemotes({
 			fs,
-			filepath: workspaceRoot ?? process.cwd(),
-		}),
-	})
-	const gitOrigin = parseOrigin({ remotes })
-	return gitOrigin
+			dir: await raw.findRoot({
+				fs,
+				filepath: workspaceRoot ?? process.cwd(),
+			}),
+		})
+		const gitOrigin = parseOrigin({ remotes })
+		return gitOrigin
+	} catch (e) {
+		console.warn(e)
```

Comment:
I would remove the console.warn. There is nothing "bad" going on if a directory has not git repo.